<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transactionsWindow</class>
 <widget class="QDialog" name="transactionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: rgb(255,255,255);
border-radius:40px;
border: 2px solid black;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Add new transaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>390</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#submitButton{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
	color: rgb(255,255,255);
}

QWidget#submitButton:hover{
	background-color: rgb(195, 195, 195);
	border-radius:20px;
	color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="amontWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#amontWidget{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
}</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="amountEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>155</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox#amountEdit {
background-color: transparent;
border: none;
outline: none;
color: rgb(255, 255, 255);
}
QDoubleSpinBox#amountEdit:drop-down {
background-color: transparent;
border: none;
color: rgb(255, 255, 255);
outline: none;
}
QDoubleSpinBox#amountEdit:down-arrow {
	image: url(:/new/prefix1/icons/chevron-down.svg);
}

QDoubleSpinBox#amountEdit:up-arrow {
	image: url(:/new/prefix1/icons/chevron-up.svg);
}</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>9999999999.989999771118164</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QWidget" name="dateWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#dateWidget{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>315</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#typeEdit{
background-color: transparent;
border: none;
outline: none;
	color: rgb(255, 255, 255);
}
QComboBox#typeEdit:drop-down {
background-color: transparent;
border: none;
	color: rgb(255, 255, 255);
outline: none;
}
QComboBox#typeEdit:down-arrow {
	image: url(:/new/prefix1/icons/chevron-down.svg);
}</string>
    </property>
    <item>
     <property name="text">
      <string>Income</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Expense</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="typeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#typeWidget{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
}</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>235</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateTimeEdit#dateEdit{
background-color: transparent;
border: none;
outline: none;
color: rgb(255, 255, 255);
}
QDateTimeEdit#dateEdit:drop-down {
background-color: transparent;
border: none;
color: rgb(255, 255, 255);
outline: none;
}

QDateTimeEdit#dateEdit:down-arrow {
	image: url(:/new/prefix1/icons/chevron-down.svg);
}

QDateTimeEdit#dateEdit:up-arrow {
	image: url(:/new/prefix1/icons/chevron-up.svg);
}</string>
    </property>
    <property name="date">
     <date>
      <year>2023</year>
      <month>5</month>
      <day>28</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2020</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>typeWidget</zorder>
   <zorder>amontWidget</zorder>
   <zorder>dateWidget</zorder>
   <zorder>submitButton</zorder>
   <zorder>label</zorder>
   <zorder>amountEdit</zorder>
   <zorder>typeEdit</zorder>
   <zorder>dateEdit</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
